ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -456,6 +462,7 @@ private[state] object Conversions {
           buildStatus(
             Code.ABORTED,
             s"Invalid ledger time: ${rejection.getDetails}",
+            Map(GrpcStatuses.DefiniteAnswerKey -> "false"),
```

Comment:
Why isn't it definite? Also, what's the source of this mapping?

Reply:
Similarly, no source. We have to come up with the values, as discussed with Miklos and Samir on Slack.

Reply:
Based on my current understanding, we can only tell that the `DUPLICATE_COMMAND` is a definite answer. @andreaslochbihler-da, is it correct?

Reply:
The rule of thumb is as follows: If the committer has updated the command dedup key with this transaction, then it can be a definite answer. Otherwise it is never a definite answer. AFAIK a `DUPLICATE_COMMAND` result never updates the command dedup key, so it's always indefinite.

Reply:
Thanks, @andreaslochbihler-da. I needed some additional explanation from @miklos-da and I think I get it now. We think that the only place where we'll be able to set the `definite_answer` to true is the rank-based command deduplication rejection (future use). Do you know any other example?

Reply:
No. That's what `definite_answer` was designed for. When we start to add rank-based deduplication logic to other commands (e.g., party migration), we'll probably use the same concept there as well.